packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -710,12 +723,20 @@ export class RestApi extends RestApiBase {
    * Import an existing RestApi that can be configured with additional Methods and Resources.
    */
   public static fromRestApiAttributes(scope: Construct, id: string, attrs: RestApiAttributes): IRestApi {
+    const restApiName = attrs.restApiName ?? '';
```

Comment:
why do we need to assign it `''`? Line 684 of this file states that the default is `ID of the RestApi construct.` This is confusing since we assign it to the empty string here instead.